packages/oidc/src/get-vercel-oidc-token.ts

```diff
@@ -22,7 +26,29 @@ import { getContext } from './get-context';
  * ```
  */
 export async function getVercelOidcToken(): Promise<string> {
-  return getVercelOidcTokenSync();
+  let token = '';
+  let err: any;
+
+  try {
+    token = getVercelOidcTokenSync();
+  } catch (error) {
+    err = error;
+  }
+
+  try {
+    const { getTokenPayload, isExpired } = await import('./token-util');
+    const { refreshToken } = await import('./token');
```

Comment:
Nit: could these be parallelized?
```suggestion
    const [
      { getTokenPayload, isExpired },
      { refreshToken },
    ] = await Promise.all([
      import('./token-util'),
      import('./token'),
    ]);
```